app/components/popover_component.html.erb

```diff
@@ -0,0 +1,3 @@
+<a tabindex="0" data-controller="popover" data-bs-placement="top" data-bs-content="<%= text %>">
```

Comment:
Could you wrap this anchor with `sup` to shrink the size and make it superscript?

Reply:
I have changes that adjust the size and color in #1090, however the designs do not call for superscript.